pytorch_lightning/callbacks/quantization.py

```diff
@@ -82,60 +82,65 @@ def _recursive_hasattr(obj: Any, attribs: str, state: bool = True) -> bool:
 
 
 class QuantizationAwareTraining(Callback):
-    OBSERVER_TYPES = ('histogram', 'average')
+    """
```

Comment:
Did you make any changes to the docstring or just moved it?

Reply:
just adding the one new argument